src/content/docs/en/reference/api-reference.mdx

```diff
@@ -1028,6 +1028,57 @@ const { Content, headings, remarkPluginFrontmatter } = await entry.render();
 
 [See the `Content Collection` guide](/en/guides/content-collections/#rendering-content-to-html) for example usage.
 
+### Other Content Collection Types
+
+The `astro:content` module also exports the following types for use in your Astro project:
+
+#### `CollectionKey`
+
+An union of string of all collection names defined in your `src/content/config.*` file. This type can be useful when defining a generic function that accepts any collection name.
+
+```ts
+import type { CollectionKey, getCollection } from 'astro:content';
+
+async function getCollection(collection: CollectionKey) {
+  return getCollection(collection);
+}
+```
+
+#### `ContentCollectionKey`
+
+An union of string of all the names of `type: 'content'` collections defined in your `src/content/config.*` file.
+
+#### `DataCollectionKey`
+
+An union of string of all the names of `type: 'data'` collection defined in your `src/content/config.*` file.
+
+#### `SchemaContext`
+
+The `context` object that `defineCollection` uses for the function shape of `schema`. This type can be useful when building reusable schemas for multiple collections.
+
+This includes the following properties:
+
+- `image` - The `image()` schema helper
```

Comment:
```suggestion
- `image` - The `image()` schema helper that... (allows you to.../ enforces.../types.../defines...)
```
I feel like this maybe wants a bit more here?

Reply:
```suggestion
- `image` - The `image()` schema helper that allows you [to use local images in Content Collections](/en/guides/images/#images-in-content-collections)
```